layouts/Error.tsx

```diff
@@ -0,0 +1,81 @@
+// CSS
+import styles from '../styles/layouts/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Footer from '@/components/Footer';
+
+const test_config_url = 'testing.com';
+
+export default function ErrorLayout({
+  children,
+}:{
+  children: React.ReactNode
+}) {
+  const [allowReport, setAllowReport] = useState(false);
+  useEffect(() => {
+    // If the user is coming from a broken link on our site, they can report an issue
+    if (document.referrer.includes('localhost:3000' || document.referrer.startsWith(test_config_url))){
+     setAllowReport(true);
+    }
+    // If not, they don't need to report an issue
+    else{
+      setAllowReport(false);
+    }
+  }, []);
+
+  return(
+    <section className={styles.container}>
+      <NavBar currentPage='Home' />
+      <main className={styles.container}>
+        <div className={styles.page}>
+          <div className={styles.pageContent}>
+            <h1>
+            Oops!
+            </h1>
+            {/* Display passed error message */}
+            {children}
+            {/* Actions the user can take */}
+            {allowReport && displayButtons()}
+            {!allowReport && displayHomeButton()}
+          </div>
+        </div>
+      </main>
+      <Footer/>
+    </section>
+  );
+}
+
+function displayHomeButton(){
+  // Allow user to return to home
+  const returnHref = '/';
+  return (
+    <>
+      <Button 
+        type={ButtonType.LIGHT}
+        href={returnHref}
+        className={styles.button}
```

Comment:
Because the left is what it looked like without styling, I'll see where the cascading is messing it up 
![image](https://github.com/user-attachments/assets/e12d6f58-f376-477c-ab39-1a66de51281e)

Reply:
Can you double check the default styles in the `button` module or if you need to add another style variant add it, I think instead of relying on external styling and cascading it might be better if we make the button component handle it internally.